### PR TITLE
4.1 1558

### LINKED ISSUE #1588: TTL: when enabling the TTL functionality the index only gets added to the default DB
## Expected Behavior (Mandatory)

Expected to have the TTL index created on those databases that didn't already have the index.

## Actual Behavior (Mandatory)

Any additional databases/graphs created using CREATE DATABASE don't have the TTL index added even when the image is restarted.

## How to Reproduce the Problem

1. Start a docker instance with apoc.ttl.enabled set to true
1. Create an additional database
1. Restart the docker instance

### Simple Dataset (where it's possibile)

```
:use neo4j
CALL db.indexes();
:use system
CREATE DATABASE missingttl;
:use missingttl
CALL db.indexes();
```
Reboot the instance and reconnect
```
:use missingttl
CALL db.indexes();
```

### Steps (Mandatory)

  1. See above

## Screenshots (where it's possibile)

nil

## Specifications (Mandatory)

nil

Currently used versions

### Versions

  - OS: Ubuntu 19.04
  - Docker: 19.03
  - Neo4j: 4.1.0
  - Neo4j-Apoc: 4.1.0.0

### PR DESCRIPTION
**Pull request by lqst**
Fixes #1588 

One sentence summary of the change.

Updated TTL documentation

A brief list of proposed changes in order to fix the issue:

  - TTL documentation now mentions creating new database

### CODE REVIEW

File: docs/asciidoc/graph-updates/ttl.adoc [hunk 1]
```diff
@@ -79,15 +79,15 @@ image::play-movies.png[title="Movies Graph Visualization"]
 [[ttl-expireAt]]
 === Expire node(s) at specified time
 
-The `apoc.ttl.expireAtInstant` procedure deletes a node or group of nodes after the datetime specified.
+The `apoc.ttl.expire` procedure deletes a node or group of nodes after the datetime specified.
```
Comment (conker84) [inline docs/asciidoc/graph-updates/ttl.adoc:82]: This is a breaking change, I think we need to guarantee the backwards compatibility.

Comment (lqst) [reply docs/asciidoc/graph-updates/ttl.adoc:82]: I am not sure I get it, there is no procedure expireAtInstant.

Comment (conker84) [reply docs/asciidoc/graph-updates/ttl.adoc:82]: Oh sorry I missed that, thank you!

**Comment by conker84** [comment]
@lqst can you please do a rebase?

**Comment by conker84** [comment]
@lqst ping

**Comment by conker84** [comment]
@lqst ping :)

File: docs/asciidoc/graph-updates/ttl.adoc (hunk 1 above)
Comment (lqst) [reply docs/asciidoc/graph-updates/ttl.adoc]: This has been fixed already in 4.1. Removing from this pr.

**Pull request merged**